src/Tokens.php

```diff
@@ -7,7 +7,7 @@ class Tokens
     public function isValid(string $namespace, int $userId, string $token) : bool
     {
         $currentTime = time();
-        $storedTime = get_user_meta($userId, '2fa_'.$namespace.'_temporary_token_time', true);
+        $storedTime = (int)get_user_meta($userId, '2fa_'.$namespace.'_temporary_token_time', true);
```

Comment:
Should the unit test be updated to account for what happens if `get_user_meta()` does not return an integer?

Reply:
Good point. Done.